docs/content/en/docs/getting-started/overview.md

```diff
@@ -4,435 +4,71 @@ linkTitle: "Overview"
 weight: 1
 aliases:
     /docs/concepts/clusterworkflow
-date: 2017-01-05
+date: 2023-08-11
 description: >
-  Explanation of the process of creating an EKS Anywhere cluster
+  Overview of the EKS Anywhere cluster creation process
 ---
 
-Each EKS Anywhere cluster is built from a cluster specification file, with the structure of the configuration file based on the target provider for the cluster.
-Currently, Bare Metal, CloudStack, Nutanix, Snow, and VMware vSphere are the recommended providers for supported EKS Anywhere clusters.
-Docker is available as an unsupported provider.
-We step through the cluster creation workflow for Bare Metal, vSphere, and Nutanix providers, here.
+#### Overview
```

Comment:
I think you can cut this heading. It should already say "Overview" at the top of the page, followed by the description.